README.md

```diff
@@ -105,3 +105,8 @@ In combination with the above instructions, add the pref `shieldStudy.logLevel=a
 ```shell
 npm start -- -f Nightly --pref=extensions.cookie-restrictions_shield_mozilla_org.test.variationName=CookiesBlocked --pref=shieldStudy.logLevel=all
 ```
+
+### After Study Survey
+
+There is a breadcrumb pref at `extensions.cookie-restrictions-shield_mozilla_org.wasEnabled`
```

Comment:
Should this be `cookie-restrictions_shield_mozilla_org` (per #43)?

https://github.com/mozilla/CookieRestrictionsShield/blob/c85d2216740965b47ea2caf59fd162c8e21925c7/test/functional/0-setup.js#L54

Or, not sure if there is a WebExt way to make these prefs shorter and we can just use `extensions.cookie-restrictions` without needing the whole "shield_mozilla_org" namespace.

Reply:
in this case we can name it whatever we like, I'll shorten it for convenience